todo_app/data/trello_items.py

```diff
@@ -0,0 +1,120 @@
+import requests, json, os
+
+trello_key = os.environ.get('TRELLO_KEY')
+trello_token = os.environ.get('TRELLO_TOKEN')
+trello_board = os.environ.get('TRELLO_BOARD')
+
+
+def get_lists():
+    """
+    Fetches all the lists for the board from trello.
+
+    Returns:
+        dictionary: key value pairs of board id and board name, as well as board name and id (for reverse lookup)
+    """    
+    lists = {}
+    url = 'https://www.trello.com/1/boards/{0}/lists?key={1}&token={2}'.format(trello_board, trello_key, trello_token)
+    resp = requests.get(url)
+    json_lists = json.loads(resp.text) 
+    for json_list in json_lists:
+        # dual purpose dictionary. keyed on name and id for reverse lookups
+        lists[json_list['id']] = json_list['name']
+        lists[json_list['name']] = json_list['id']
+    return lists
+
+
+def get_items():
+    """
+    Fetches all cards from trello.
+
+    Returns:
+        list: The list of cards.
+    """
+    lists = get_lists()
+
+    url = 'https://www.trello.com/1/boards/{0}/cards?key={1}&token={2}'.format(trello_board, trello_key, trello_token)
+    resp = requests.get(url)
+    items = json.loads(resp.text)
+
+    # enrich the output by appending the list name to each card, based on its list id
+    for item in items:
+        item['list_name'] = lists[item['idList']]
+
```

Comment:
Part 5 of the exercise was asking for a relatively simple data class. A way of defining what an "item" actually looks like, as dictionaries are entirely mutable and you have no obvious place in your codebase to go to and see "ah right, an item has properties x, y, z". And then you also aren't tied to Trello's data structure as much. Only one place needs to know how to map from Trello's data to your Item class, and then the rest of your code can confidently use that class.

i.e. define this somewhere
```
class Item:
    def __init__(self, id, name, status):
        self.id = id
        self.name = name
        self.status = status
```

And then here you can do:
```
    mapped_items = []
    for item in items:
        status = lists[item['idList']]
        mapped_item = Item(item['id'], item['name'], status)
        mapped_items.append(mapped_item)
```
Things may need renaming to work with your app. E.g. if index.html uses `list_name` rather than `status`.

And then return `my_items` instead of `items`.

The POST endpoints don't need to worry about Item because they don't deal with a whole item.